### PR TITLE
Apply the DRY principle and add better formatting

### PR DESCRIPTION
**Pull request by JalalHamed**
### 2 big objects
`enToFaDictionary` and `faToEnDictionary` are 2 big objects. With their **keys** and **values** being the exact reversed version of each other! So by reversing one (in this case `enToFaDictionary`) and assigning it to the other, we still get the same result only with much fewer lines of code (63 to be exact).
**Side Note**: This could be accomplished using `Array.reduce` method and more advance ES6 features as well. A `for` loop is the simplest one.

### using quotes for object keys
unless you want to use a key that’s not a valid JavaScript identifier, wrapping object keys in quotes is not necessary and it doesn't make a difference.
**Side Note**: the [JSON data exchange format](https://www.json.org/json-en.html) _does_ require double quotes around identifiers (and does _not_ allow single quotes).

### semicolons
Some lines were missing it, my prettier automatically added it to them.

### CODE REVIEW

File: popup.js [hunk 1]
```diff
@@ -8,175 +8,117 @@ let translatorLanguage = 'fa';
...
+const translator =
+    (lang = 'fa') =>
+    str =>
+        str
+            .trim()
+            .split('')
+            .map(s => {
+                if (s === ' ') return ' ';
+                if (lang === 'fa') {
+                    return enToFaDictionary.hasOwnProperty(s)
```
Comment (realamirhe) [inline popup.js]: cause every character in the dictionary has the truthy value, what about changing it to something like this.

```js
if (lang === 'fa') return enToFaDictionary[s] || faToEnDictionary[s] || s

return faToEnDictionary[s] || enToFaDictionary[s] || s
```

Comment (hadizz) [reply popup.js]: I agree with that. it's cleaner.👌🏻

**Comment by hadizz** [comment]
Hi @JalalHamed and thanks for contributing❤️

You have made amazing changes and improved the performance of the code🚀

For 2 big objects part:
I decided to save two objects because I thought that "for loop" will cost me more than storing them in memory. but for now, I think it would work fine and not get performance issues. and of course, it follows the DRY principle.

Please review this thread:
https://github.com/hadizz/persian-typo-mistake-extension/pull/1#discussion_r785133590

For the prettier part:
Could you please add your prettier configs to the project? so that we have some convention between contributors.

File: popup.js (hunk 1 above)
Comment (hadizz) [reply popup.js]: Sorry @amirHossein-Ebrahimi, I forget to say Hi :), and thanks for reviewing🙏🏻

**Comment by JalalHamed** [comment]
For sure. I'm on it.

**Comment by hadizz** [comment]
Thanks for helping🙏🏻 I checked the pull request and I think it's ready to be merged.

**Pull request merged**